lighthouse-core/lib/url-shim.js

```diff
@@ -23,7 +23,7 @@
 
 /* global self */
 
-const ELLIPSIS = '\u2026';
+const Util = require('../report/v2/renderer/util.js');
```

Comment:
should `getUrlDisplayName` move to a utility file in `lighthouse-core/lib/`? Seems weird for url-shim to reach into report rendering to get it, while the report generator already has to go out for `lib/file-namer.js`